hack/release/sync-repo-labels.sh

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env bash
+
+readonly PROGNAME=$(basename "$0")
+readonly HERE=$(cd "$(dirname "$0")" && pwd)
+readonly REPO=$(cd "${HERE}/../.." && pwd)
+
+readonly TOKEN=${TOKEN:-}
+
+readonly DOCKER=${DOCKER:-docker}
+readonly LABELS=${LABELS:-${REPO}/site/_data/github-labels.yaml}
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+labelsync() {
+    $DOCKER run \
+        --rm \
+        --interactive \
+        --tty \
+        --env GITHUB_TOKEN \
+        --volume "${REPO}:/$(basename "${REPO}")" \
+        gcr.io/k8s-prow/label_sync:latest \
```

Comment:
I still don't like using latest for anything. Also not 100% sure about what this command does and how it will affect all the issues.

Reply:
This is the tool that comes out of Kubernetes test-infra and applies labels to all the Kubernetes repositories. Not sure that they have a fully engineered release for it; I think they just just latest too.